src/test/java/com/amihaiemil/eoyaml/spec/SpecExamplesITCase.java

```diff
@@ -41,21 +41,22 @@
  * @author Aislan Nadrowski (aislan.nadrowski@gmail.com)
  * @version $Id$
  * @since 4.3.6
- * 
- * @todo #238:30min Add more tests to cover all the examples from the Yaml 1.2 
- *  Specification. You can have a look at PR 335 to see how other examples have 
- *  been tested. Don't forget that you do not have to write all the tests in 
- *  this task -- write 1 or 2 tests, leave a todo (puzzle) and the work will 
- *  continue in a future ticket.
+ *
+ * @todo #336:30min Add more tests to cover all the examples from the Yaml 1.2
+ * Specification. You can have a look at PR 335 to see how other examples have
+ * been tested. Don't forget that you do not have to write all the tests in this
+ * task -- write 1 or 2 tests, leave a todo (puzzle) and the work will continue
+ * in a future ticket.
```

Comment:
@ANadrowski Can you pls fix this puzzle's indentation? Lines 2...n should start with a space (they should be aligned with the ``t``, not with the ``@``. Otherwise, 0pdd will complain...

Reply:
@amihaiemil I am sorry, the IDE was not well configured. Please see if now it is ok. Thank you.